### DISCUSSION TITLE
ActionMailer::MailDeliveryJob stuck

### DISCUSSION THREAD

**Discussion by greyidol** [Retrospring/retrospring#1008] [Q&A] [answered]
I have this in my config/environments/production.rb:
```ruby
  config.action_mailer.smtp_settings = {
    port: 587,
    address: 'smtp.purelymail.com',
    user_name: 'ask@[redacted]',
    password: '[redacted]',
    domain: '[redacted]',
    authentication: :plain,
    ca_file: '/etc/ssl/certs/ca-certificates.crt',
    openssl_verify_mode: :none,
    enable_starttls: nil,
    enable_starttls_auto: 'true',
  }
```
But when I check the sidekiq panel, I see that the `mailers` queue has a `ActionMailer::MailDeliveryJob` stuck in the queue for 45 minutes now. How can I debug this?

**Comment by nilsding** [comment]
honestly?  no idea -- we use `sendmail`.  but if a job is stuck in the queue it generally sounds like sidekiq isn't configured properly to work on the queues, it should show up on the "Busy" page of the web UI.

anyway, closing this issue because this is not related to Retrospring itself.  support for getting self-hosted environments up and running is out of scope for this issue tracker.

**Comment by greyidol** [comment]
Is it in scope for the [Discussions](https://github.com/Retrospring/retrospring/discussions) page?

**Comment by nilsding** [comment]
ah right, i forgot github has a forum now... but yes, it would be more fitting there.

**Comment by greyidol** [comment, marked as answer]
Turns out the queue was stuck because I had set a redis namespace for sidekiq. Removing the namespace and using a separate database number in the redis path got the job to run again.
